### PR TITLE
Onboard managed-upgrade-operator to CI

### PR DESCRIPTION
**Pull request by ravitri**
Onboarding managed-upgrade-operator as per https://issues.redhat.com/browse/OSD-3781

### CODE REVIEW

File: ci-operator/config/openshift/managed-upgrade-operator/openshift-managed-upgrade-operator-master.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: "4.4"
+    namespace: ocp
+    tag: base
```
Comment (jharrington22) [inline ci-operator/config/openshift/managed-upgrade-operator/openshift-managed-upgrade-operator-master.yaml:6]: @ravitri we should change this to the `ubi` image and tag `8`

Comment (ravitri) [reply ci-operator/config/openshift/managed-upgrade-operator/openshift-managed-upgrade-operator-master.yaml]: Done.. Thanks a lot for this :)

**Pull request merged**